tl.tl

```diff
@@ -1685,6 +1685,10 @@ do
          i, e1 = parse_literal(ps, i)
       end
 
+      if not e1 then
+         fail(ps, i, "expected an expression")
+         return i
+      end
```

Comment:
if `e1` doesn't exist, another error must have been already through in all paths leading here, right? I've been trying to not output repeated errors (a recent commit got rid of a few recently)